content/reference/glossary/bunt.md

```diff
@@ -0,0 +1,17 @@
++++
+title = "Bowl"
```

Comment:
Bunt and Bowl are swapped files.

Reply:
While I'm thinking about it, the only place "clam" occurs is https://developers.urbit.org/reference/hoon/stdlib/4n#soft.  But we should probably have an entry for it since it is obscure but does get said from time to time.